website/_includes/head_custom.html

```diff
@@ -1,18 +1,19 @@
-<!-- Generated using https://realfavicongenerator.net. -->
+<!-- Generated from https://realfavicongenerator.net/ -->
```

Comment:
```suggestion
<!-- Generated using https://realfavicongenerator.net. -->
```
Did you make this changes? Or not? To me the original statement makes more sense.

Reply:
See edit in https://github.com/PicnicSupermarket/error-prone-support/pull/408#issuecomment-1346564248. Likely a syncing issue between branches. I'd favor `using` to.